lab/datamanagement/tests/common.py

```diff
@@ -28,9 +28,9 @@ def _create_experiment(timeslot_dts=None) -> Experiment:
 
     exp = Experiment.objects.create(
         name='test',
-        leader=_get_or_create_leader(),
```

Comment:
Perhaps this function should be made public so the "_" should be stripped of the function, as we are importing it. It was already imported as _get_or_create_leader, so

Reply:
I assume the leading underscore is there because these are test functions (that's code inherited from the adult system). I don't mind changing it, but then I don't want to change all these helper functions in the current PR.